Jenkinsfile

```diff
@@ -139,23 +140,24 @@ pipeline {
             }
```

Comment:
It won't let me comment directly on L133 (just above this), so I'm putting it here.

We should be running the header tests on every place where you said we'd be testing (the three places you've highlighted). This will catch compiler incompatibilities and some header include incompatibilities. We should be passing this on all the compiler / platforms you've listed.

Reply:
I'm just doing a review and checking what you've said against what you've done. Here, it's just come up short. 

So... please add it so it matches your top-level PR comment? Or... update your top-level PR comment with this special case? I think it makes more sense to add it if that's what we're claiming we're testing.

Reply:
The top-level note is referring to the unit tests in most places that lack a qualifier. I can add that explicitly to the "now we test" section. I'm not sure how the header include order tests would break on just one compiler but we can address that

Reply:
Thanks for updating that in the pull request. Since this pull request doesn't fully fix the issue, can you do these things:
1. update issue #943 appropriately so it's scoped to what you've fixed.
2. create a new issue for the remaining items that weren't addressed here so we know to go and fix them?

I'm happy getting this in as an partial fix to the overall issue, but we need to note that it's not complete.

Reply:
Are you just saying to liberally sprinkle the phrase "unit tests" all over #943? Are the remaining issues you're talking about the sysadmin stuff on the Xenial box? We haven't typically created math repo issues for that kind of thing before.